com.project_rk.launcher.yml

```diff
@@ -0,0 +1,232 @@
+    version: *runtime-version
+    no-autodownload: true
+    autodelete: false
+
+  org.freedesktop.Platform.ffmpeg_full.i386:
+    directory: lib32/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+    no-autodownload: true
+    autodelete: false
+
+  com.valvesoftware.Steam.CompatibilityTool:
+    subdirectories: true
+    directory: share/steam/compatibilitytools.d
+    version: stable
+    versions: stable;beta;test
+    no-autodownload: true
+    autodelete: true
+
+  com.valvesoftware.Steam.Utility:
+    subdirectories: true
+    directory: utils
+    version: stable
+    versions: stable;beta;test
+    add-ld-path: lib
+    merge-dirs: bin;lib/python3.10/site-packages;share/vulkan/explicit_layer.d;share/vulkan/implicit_layer.d;share/steam/compatibilitytools.d;
+    no-autodownload: true
+    autodelete: true
+    
+modules:
+  # Drivers structure
+  - name: platform-bootstrap
+    buildsystem: simple
+    build-commands:
+      - |
+        set -e
+        mkdir -p /app/bin
+        mkdir -p /app/lib/i386-linux-gnu
+        mkdir -p /app/lib/i386-linux-gnu/GL
+        mkdir -p /app/lib/i386-linux-gnu/dri/intel-vaapi-driver
+        mkdir -p /app/lib/debug/lib/i386-linux-gnu
+        mkdir -p /app/lib/debug/lib/i386-linux-gnu/GL
+        install -Dm644 -t /app/etc ld.so.conf
+        mkdir -p /app/lib{,32}/ffmpeg
+        mkdir -p /app/share/steam/compatibilitytools.d
+        mkdir -p /app/utils /app/share/vulkan
+        ln -srv /app/{utils/,}share/vulkan/explicit_layer.d
+        ln -srv /app/{utils/,}share/vulkan/implicit_layer.d
+        mkdir -p /app/links/lib
+        ln -srv /app/lib /app/links/lib/x86_64-linux-gnu
+        ln -srv /app/lib32 /app/links/lib/i386-linux-gnu
+    sources:
+      - type: inline
+        dest-filename: ld.so.conf
+        contents: |
+          # We just make any GL32 extension have higher priority
+          include /run/flatpak/ld.so.conf.d/app-*-org.freedesktop.Platform.GL32.*.conf
+          /app/lib32
+          /app/lib/i386-linux-gnu
+          /lib64
+
+  - name: umu
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 umu-run $FLATPAK_DEST/bin/umu-run
+      - install -Dm755 umu_run.py $FLATPAK_DEST/bin/umu-run.py
+    sources:
+      - type: archive
+        url: https://github.com/Open-Wine-Components/umu-launcher/releases/download/1.1.3/Zipapp.zip
+        sha256: bb26f7bce5291076c056dca9d0ded0bc0eb0b246d74e73b16e63808953b0f995
+
+  # Project RK Scripts
+  - name: project-rk
+    buildsystem: simple
+    build-commands:
+      - install -Dm 755 project-rk.sh ${FLATPAK_DEST}/bin/project-rk
+      - install -Dm 755 dgVoodooCpl.sh ${FLATPAK_DEST}/bin/dgVoodooCpl
+      - install -Dm 755 dgVoodooCpl.exe ${FLATPAK_DEST}/bin/dgVoodooCpl.exe
+    sources:
+      - type: file
+        path: bin/project-rk.sh
+      - type: file
+        path: bin/dgVoodooCpl.sh # Added for dgvoodoo control panel config support
+      - type: file
+        path: bin/dgVoodooCpl.exe
```

Comment:
so you removed the exe from here, but where is this going to come from now?

Reply:
our upstream updated files, I commented earlier about the original intent and why I would be open to removing this part. We can package this by default, it's just not in our current release branch. We had intended to provide this as sort of a tech-preview optional feature ahead of packing it in our auto-updated files.

This flatpak just helps stand up the wineprefix and containerized runtime environment for our launcher; like other apps like discord that self-update with a cache (mutable data), ours just keeps the prefix hydrated.

Reply:
ok then remove this source from the manifest.

Reply:
done!